app/i18n/zh-cn/sub.php

```diff
@@ -178,10 +178,10 @@
 		'max_http_redir' => '最大 HTTP 重定向',
 		'max_http_redir_help' => '设置为 0 或留空以禁用，-1 表示无限重定向',
 		'method' => array(
-			'_' => 'HTTP Method',	// TODO
+			'_' => 'HTTP 方式',
 		),
-		'method_help' => 'The POST payload has automatic support for <code>application/x-www-form-urlencoded</code> and <code>application/json</code>',	// TODO
-		'method_postparams' => 'Payload for POST',	// TODO
+		'method_help' => 'POST 荷载自动支持 <code>application/x-www-form-urlencoded</code> 和 <code>application/json</code>',
+		'method_postparams' => 'POST 荷载',
```

Comment:
One other thing. Perhaps `method_postparams` should be renamed to `method_payload`?

Reply:
Params is clearer than payload imo (unless there are no multiple params involved but only a single payload).

Reply:
https://github.com/FreshRSS/FreshRSS/blob/9c97d8ca729e3cfb067445c0d3c9ad8284132aeb/app/views/subscription/add.phtml#L333
https://github.com/FreshRSS/FreshRSS/blob/350edf398c55b472e19a3017de9b4d2d3420b9e4/app/Controllers/feedController.php#L184
https://github.com/FreshRSS/FreshRSS/blob/350edf398c55b472e19a3017de9b4d2d3420b9e4/app/Controllers/feedController.php#L209

How about `method_postfields`?